docs/authentication/social-connections/twitch.mdx

```diff
@@ -29,10 +29,12 @@ First, you need to register a new OAuth Twitch app at the [Twitch Developers Con
 
 ![Creating an OAuth Twitch app](/docs/images/authentication-providers/twitch/b23861d81906f6e2feedd314c520dc3b0d744d70-1891x757.png)
 
-Set a name and a category for your new application. You also need to add the **OAuth Redirect URLs.** In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable Twitch. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **OAuth Redirect URLs** input and click create.
+Set a name and a category for your new application. You also need to add the **OAuth Redirect URLs.**
+
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **Twitch**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Paste the value into the **OAuth Redirect URLs** input and click create.
 
 Once all the above are complete, copy the **Client ID** and **Client Secret.** Go back to the Clerk Dashboard and paste them into the respective fields.
 
 ![Obtaining the Client ID and Client secret](/docs/images/authentication-providers/twitch/b13fd18936b6cfd4012784fbb044cd7a27afb5c1-1270x729.png)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with Twitch is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with Twitch is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with Twitch is now configured for your instance.
```